app/webpacker/styles/_govuk_overrides.scss

```diff
@@ -1,3 +1,9 @@
 .govuk-notification-banner__content .app-invalid-answer__list a:link {
   font-weight: bold;
 }
+
+.govuk-error-summary, .govuk-error-message {
+  .no-wrap {
+    white-space: nowrap;
```

Comment:
I think this could be useful in places other than the error summaries.

Reply:
Also — should it be name spaced?

Reply:
I've un-nested it (if that even makes sense). I think organisation of our css is a good candidate for a fix it Friday to workout as a team so will card up a ticket to look into this.